api/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
```

Comment:
Why are you importing this User model? I believe it is enough to create class Users(models.Model): like you do in line 12.